src/schema/v2/artist/helpers.ts

```diff
@@ -72,12 +85,13 @@ export const getArtistInsights = (artist) => {
   ][]
 
   const insights = mappings.map((mapping) => {
-    const [kind, { getDescription, getEntities, label }] = mapping
+    const [kind, { getDescription, getEntities, getLabel }] = mapping
```

Comment:
Given enough time everything in javascript ends up being a function 😝

Reply:
you're a big fan of mapping functions! Love that!